README.md

```diff
@@ -66,12 +70,15 @@ open ~/.zprofile
 ```
 
 Then in the text editor add :
+
 ```bash
 export CLIENT_ID=hmpps-test-id
 export CLIENT_SECRET='.01EAWS33333OGu=Q+%aA%tn1T-UB+xAewSf3*Qww,4S,Vk>s3372j)AaTtO2'
 ```
+
 The client id and secret above are not a real ones, to get the id and secret you must run the following command.
-Please make sure the CLIENT_SECRET value is wrapped by 'CLIENT_SECRET_VALUE' 
+Please make sure the CLIENT_SECRET value is wrapped by 'CLIENT_SECRET_VALUE'
+
 ```bash
 kubectl -n visit-someone-in-prison-backend-svc-staging  get secrets hmpps-prison-visits-testing-helper-api -o json | jq '.data | map_values(@base64d)'
```

Comment:
Shall we remove this command?

Reply:
Narr no damage,... as this command is standard stuff...